pkg/ddevapp/magento.go

```diff
@@ -23,7 +23,7 @@ func isMagentoApp(app *DdevApp) bool {
 
 // isMagento2App returns true if the app is of type magento2
 func isMagento2App(app *DdevApp) bool {
-	ism2, err := fileutil.FgrepStringInFile(filepath.Join(app.AppRoot, app.Docroot, "..", "SECURITY.md"), `https://hackerone.com/magento`)
+	ism2, err := fileutil.FgrepStringInFile(filepath.Join(app.AppRoot, app.Docroot, "..", "SECURITY.md"), `https://hackerone.com/`)
```

Comment:
I am not a fan of looking for the string `https://hackerone.com/` in a `SECURITY.md` file. That is too generic.

If we want to stick with that file, we could search for the string `magento` which is present multiple times in that file: https://github.com/magento/magento2/blob/2.4.6-p4/SECURITY.md?plain=1

Reply:
It's your PR! It can use anything reasonable to detect, but should work with older and newer magento2 and be expected to be relatively unique.